app/cmd.go

```diff
@@ -515,6 +520,12 @@ func getConfig() (appConfig, error) {
 		return cfg, validation.NewFieldError("db-url", "is required")
 	}
 
+	if cfg.TLSListenAddr != "" {
+		if cfg.TLSCertFile == "" && cfg.TLSKeyFile == "" && cfg.TLSCert == "" && cfg.TLSKey == "" {
```

Comment:
Hmm. I'm wondering if `appConfig` should just have `*tls.Config` as a single field, and handle reading/loading files here (or in it's own func).

In the future, that would allow the app to simply:
- bind TLS if the listen field is specified
- use `*tls.Config` directly, or default to ACME support if it's nil

Let's have a quick chat on this.